tests/lib/tools/journal-state

```diff
@@ -73,7 +79,29 @@ get_journalctl_log_from_cursor(){
     fi
 }
 
-sync_journalctl_log(){
+_sync_journalctl_log(){
     journalctl --flush || true
     journalctl --sync || true
 }
+
+if [ $# -eq 0 ]; then
+    show_help
+    exit 1
```

Comment:
Nitpick, usually tools don't fail in this case, only in the case when the arguments are actually invalid.

Reply:
done